credits.txt

```diff
@@ -157,63 +164,69 @@ contributed to Endless Sky:
   DeBlister
   DingusShingleton
   Disiuze
+  DJF113
+  DownsB
   dplepage
+  dragonmaus
   DrBlight
   dreness
   dufferzafar
   dzhu
   eflyon
-  EjoThims
```

Comment:
Also can't find anything Ejo has contributed.

Reply:
Ouch. :P

TBF I don't have any direct PRs or assets to my name yet, but I think that comes from review/edit submissions within PRs that then were squashed?

Probably the same for the others initially listed.

Reply:
This was added in #5945 - and sifting through the commits of 0.9.14, it's probably due to #5965, which was squash-merged into a commit that was technically co-authored by you.